src/components/UserCollective.js

```diff
@@ -306,7 +306,7 @@ class UserCollective extends React.Component {
 
             {this.props.message && <MessageModal message={this.props.message} />}
 
-            <CollectiveCover collective={collective} cta={cta} LoggedInUser={LoggedInUser} />
+            <CollectiveCover collective={collective} cta={cta} LoggedInUser={LoggedInUser} key={collective.slug} />
```

Comment:
Discovered this while testing out the staging site for this branch. When navigating between USER or ORGANIZATION collectives, there is the chance of a cached render that we should break with the `key` prop.

Reply:
@Betree reported a similar bug but could not reproduce. Not sure this is it, but just in case: https://github.com/opencollective/opencollective/issues/1474